tools/tertiary-analysis/sc3/sc3-macros.xml

```diff
@@ -0,0 +1,22 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+        <requirement type="package" version="0.0.1">bioconductor-sc3-scripts</requirement>
+            <yield/>
+        </requirements>
+    </xml>
+    <xml name="citations">
+        <citations>
+            <citation type="bibtex">
+@misc{githubbioconductor-sc3-scripts.git,
```

Comment:
Please add the adequate content to fields on the citation. You can export bibtex formatted text for a citation from pubmed or other providers.